banzai_nres/tests/test_extract.py

```diff
@@ -83,6 +83,39 @@ def test_extract_in_readnoise_regime(self):
             assert optimal_median_sn > 1.45 * box_median_sn
 
 
```

Comment:
This is a test that fails under the old way of extracting the wavelength solution.

It basically shifts the wavelength regions by 1, on a semi realistic image, and makes sure that the extracted wavelengths are good.